resources/js/filters.js

```diff
@@ -15,11 +15,12 @@ Vue.filter('price', function (value) {
 
 window.url = function (path = '') {
     // Transform urls starting with / into url with domain
-    if (!path.startsWith('/')) {
-        return path
+    if (path.startsWith('/')) {
+        path = (window.config.base_url || window.origin) + path
     }
 
-    return (window.config.base_url || window.origin) + path
+    // Replace any double slashes not preceded by a : with a single slash
+    return path.replaceAll(/([^:])\/\//g, '$1/')
```

Comment:
We could still pass in a relative url.
This could be problematic when passing in external urls (which this function is made to accept)
as https://www.ietf.org/rfc/rfc2396.txt defines the text between `/` are completely optional, thus it's up to the server wether `https://example.com/test//page` is the same page as `https://example.com/test/page`

I think it's best to resolve this problem where it occurs so we don't break any external routes.